### DISCUSSION TITLE
Ability to keep Popover.Dropdown element always visible in the current container

### DISCUSSION THREAD

**Discussion by vhpx** [mantinedev/mantine#3277] [Feature requests]
### What package has an issue

@mantine/core

### Describe the bug

Currently, the Popover component has a special mechanism to keep the dropdown element on the screen until the root element (which houses the Popover component) is out of sight, then the dropdown element will follow suit.

I want to know if there is any available configuration that can be added to the Popover component so that the Popover.Dropdown can always be displayed on the screen even if the root element is no longer visible (this might be because it was scrolled away from the viewport).

### What version of @mantine/hooks page do you have in package.json?

5.10.0

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

### Possible fix

Include an option for the Popover component to toggle between always following the root element and semi-follow the root element while remaining in visible sight at all times.

**Comment by rtivital** [comment]
Popover is not unmounted from the dom when you scroll – https://imgur.com/a/kBxKAR4

**Reply by vhpx** [reply]
It is indeed preserved in the dom. Is there any way to force it to be always be visible?

For example, when you scroll down on an infinitely long page, the popover dropdown will be pushed to the top but then remain visible on the screen instead of being stationary to the element with the passed ref.

**Reply by rtivital** [reply]
No, it is not supported. Add `position:sticky` to the target element instaed
